app/DoctrineMigrations/Version20260727000000.php

```
@@ -0,0 +1,48 @@
+        if (!$schema->hasTable(self::NAME)) {
+            return;
+        }
+
+        $blockExists = $this->connection->fetchOne("SELECT COUNT(*) FROM dtb_block WHERE file_name = 'faq'");
+        if ($blockExists == 0) {
+            $this->addSql("INSERT INTO dtb_block (device_type_id, block_name, file_name, create_date, update_date, use_controller, deletable, discriminator_type) VALUES (10, 'FAQ', 'faq', CURRENT_TIMESTAMP, CURRENT_TIMESTAMP, '1', '0', 'block')");
+        }
+    }
+
+    #[\Override]
+    public function down(Schema $schema): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)